content/applications/productivity/iot/config/connect.rst

```diff
@@ -28,54 +31,78 @@ Ethernet Connection
 #. Read the Pairing Code from a screen or a receipt printer connected to the IoT Box.
 
    .. image:: connect/connect04.png
+       :align: center
+       :alt: Pairing code for a wired connection.
 
-#. Input the Pairing Code and click on the Pair button.
+..
+   _Image needs to be updated.
+
+#. Input the Pairing Code on the IoT module of the database and click on the Pair button.
 
 .. note::
-   Recent changes in modern web browsers forced us to modify the connection wizard.
-   If your screen is different from the screenshots, make sure that the
-   ``iot_pairing`` module is installed.
+   Recent changes in modern web browsers forced us to modify the connection wizard. If the screen is
+   different from the screenshots, make sure that the ``iot_pairing`` module is installed.
+
+..
+   _This module doesn't exist any longer.
 
 WiFi Connection
 ===============
 
 #. Power on the IoT Box
 
-#. Copy the token
+#. Copy the token from the WiFi connection section in IoT module of the Odoo database.
 
    .. image:: connect/connect05.png
+      :align: center
+      :alt: Pairing Token for a WiFi connection.
 
-#. Connect to the IoT Box WiFi Network (make sure there is no ethernet
-   cable plugged in your computer).
+#. Connect to the IoT Box WiFi Network (make sure there is no ethernet cable plugged in the
+   computer).
 
    .. image:: connect/connect06.png
+      :align: center
+      :alt: WiFi networks available on the computer.
+
+..
+   _Image needs to be updated.
 
-#. You will be redirected to the IoT Box Homepage (if it doesn't work,
-   connect to the IP address of the box). Give a name to your IoT Box (not
-   required) and paste the token, then click on next.
+#. Upon connecting to the IoT WiFi a browser will automatically redirect to the IoT Box Homepage (if
+   it doesn't work, connect to the IP address of the box). The default address of the IoT box is
+   `localhost:8069 <http://localhost:8069/>`. Give a name to the IoT Box (not required), paste the
```

Comment:
I have some doubt on the fact that `localhost:8069` will be the address used. Would you mind double checking on the matter (with a runbot or something like this).

Reply:
@lse-odoo This was the case with an IoT box that didn't have the secured certificate triggered from the subscription. Can your team verify that this IP address is/isn't different each time? I consistently got localhost:8069

Reply:
I've removed the specifics of the URL. I'm going to resolve this conversation.